tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,21 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
+      it('#5471 PouchDB.plugin() should throw error if passed wrong type or empty object', function () {
+        try {
+          PouchDB.plugin('pouchdb-adapter-memory');
```

Comment:
have a look at http://chaijs.com/api/bdd/#method_throw. You want the test to fail if it doesn't throw.

```
expect(function() {
    PouchDB.plugin('pouchdb-adapter-memory');
}).to.throw(Error, 'Invalid plugin: object passed in is empty or not an object');
```